code/modules/language/unathi.dm

```diff
@@ -0,0 +1,20 @@
+/datum/language/unathi
+	name = "Unathi"
+	desc = "The common language of lizard-people, composed of sibilant hisses and rattles."
+	speech_verb = "hisses"
+	ask_verb = "hisses"
+	exclaim_verb = "roars"
+	key = "o"
+	flags = TONGUELESS_SPEECH
+	space_chance = 40
+	syllables = list(
+		"za", "az", "ze", "ez", "zi", "iz", "zo", "oz", "zu", "uz", "zs", "sz",
```

Comment:
First time I was opening this code, I'd hoped to see `"s", "S",` and nothing but.